static_report/src/components/ComparisonReport/CRTableList/CRTableListColumnDetails.tsx

```diff
@@ -0,0 +1,77 @@
+import { useMemo } from 'react';
+
+import { CRTableListColumnDetail } from './CRTableListColumnDetail';
+import {
+  getIconForColumnType,
+  transformAsNestedBaseTargetRecord,
+} from '../../../utils/transformers';
+import type {
+  ColumnSchema,
+  TableSchema,
+} from '../../../sdlc/single-report-schema';
+
+export function CRTableListColumnDetails({
```

Comment:
recommend call this with suffix `*List` since it is too similar to its consumed iterated children, `CRTableListColumnDetail`

Reply:
done